contrib/fuzz/unstable_call_graph.py

```diff
@@ -0,0 +1,235 @@
+    parser.add_argument(
+        "--skip-strings-file",
+        help="Path to file that contains strings that if at the start of a function name, that function is skipped",
+        type=str,
+        default=None)
+    parser.add_argument(
+        "--dump-hierarchy",
+        help="Path to json file where the hierarchy will be dumped",
+        type=str,
+        default=None)
+    parser.add_argument(
+        "--dump-unstable",
+        help="Path to file where unstable symbols will be dumped",
+        type=str,
+        default=None)
+    parser.add_argument(
+        "--no-tui",
+        help="Whether to output the hierarchy to the terminal",
+        type=bool,
+        default=False)
+
+    args = parser.parse_args()
+    
+    unstable_edges = []
+    unstable_functions = set()
+    strings_to_skip = set()
+
+    with open(args.afl_fuzzer_stats, 'r') as fuzzer_stats_file:
+        for line in fuzzer_stats_file:
+            if line.startswith('var_bytes'):
+                unstable_edges = line.split(":")[1].strip().split(" ")
+                break
+
+    with open(args.afl_edge_id_file, 'r') as edge_id_file:
+        for line in edge_id_file:
+            for edge in unstable_edges:
+                if f'edgeID={edge}' in line:
+                    function = line.strip().split(" ")[1].split("=")[1]
+                    if function not in unstable_functions:
+                        unstable_functions.add(function)
+
+    if args.dump_unstable:
+        with open(args.dump_unstable, 'w') as unstable_file:
+            for function in unstable_functions:
+                unstable_file.write(function + '\n')
+    
+    if args.skip_strings_file:
+        with open(args.skip_strings_file, 'r') as skip_file:
+            strings_to_skip = set(line.strip() for line in skip_file if line.strip())
+
+    try:
+        lines = check_output(["objdump", "-d", args.binary_file]).splitlines()
+    except:
+        exit()
+
+    G = nx.DiGraph()
+    curFunc = None
+    architecture = get_architecture(args.binary_file)
+
+    if architecture == 'x86_64':
+        call_pattern = r'^.*\bcall\s+([0-9a-zA-Z])+\s+<(.*)>$'
+    elif architecture == 'arm64':
+        call_pattern = r'^.*\bbl\s+([0-9a-zA-Z])+\s+<(.*)>$'
+    else:
+        raise ValueError("Unsupported architecture: " + architecture)
+
+    for l in lines:
+        l = l.decode('utf-8')
+        m = re.match(r'^([0-9a-zA-Z]+)\s+<(.*)>:$', l)
+        if m:
+            curFunc = m.group(2)
+            continue
+
+        if curFunc == None:
+            continue
+
+        m = re.match(call_pattern, l)
+        if m:
+            G.add_edge(curFunc, m.group(2))
+
+    print(f"Graph has {G.number_of_nodes()} nodes and {G.number_of_edges()} edges.")
+    
+    seen_edges = set()
+
+    def create_hierarchy(G, root):
+        demangled = demangle(root)
+        node = {
+            demangled: {
+                'children': {},
+                'is_unstable': root in unstable_functions,
```

Comment:
I tried using `--skip-strings-file` and I think it would be better if it simply doesn't color the skipped symbols but still shows them in the hierarchy.

```suggestion
                'is_unstable': root in unstable_functions and not any(demangled.startswith(string) for string in strings_to_skip),
```

Reply:
Do we still want the behavior where the unstable child tree doesn't get included in the hierarchy? Once a root is labled unstable we stop there